web/src/components/molecules/Common/CoreWrapper/index.tsx

```diff
@@ -29,7 +29,7 @@ const Content = styled.div`
   display: flex;
   flex-direction: column;
 
-  > :first-child {
+  > :first-of-type {
```

Comment:
Sorry I asked you not to touch this component, but instead find a way to make it work in each page's components.